core/deco.c

```diff
@@ -506,6 +506,7 @@ void add_segment(struct deco_state *ds, double pressure, const struct gasmix *ga
 	return;
```

Comment:
comment for the commit message of `7fec44ded1`.
it's best to have a single short sentence with length around 70-80 characters as the commit title.
then the commit message body should start with a new sentence.

but that's just a nitpick for the `...` ellipsis.